app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -45,23 +48,34 @@ def supports_authentication?(authtype)
     supported_auth_types.include?(authtype.to_s)
   end
 
-  def self.raw_connect(server, port, username, password, service = "Service")
-    require 'ovirt'
+  def raw_connect(server, port, username, password, service = "Service")
+    # If there is API path stored in the endpoints table and use it:
+    path = default_endpoint.path
+    unless path.blank?
+      _log.info("Using stored API path '#{path}'.")
```

Comment:
@jhernand by "inlining" I mean putting the unless clause on the same line... `_log.info(...) unless path.blank?`